specs/InfoBadge/InfoBadge-spec.md

```diff
@@ -0,0 +1,526 @@
+
+
+# Conceptual pages (How To)
+
+Badging is the least intrusive and most intuitive way to display notifications or bring focus to something within an app. An InfoBadge is a small piece of UI that can be added into an app and customized to display a number, icon, or a simple dot. **InfoBadge is built into NavigationView but can also be added as a standalone element in the XAML tree, allowing you to place InfoBadge into any control or piece of UI of your choosing.** When using an InfoBadge somewhere other than NavigationView, you will be responsible for programmatically determining when to show/dismiss the InfoBadge and where to place the InfoBadge. An InfoBadge should be used to bring the user’s focus to an area – whether that be for notifications, indicating new content, or showing an alert.
+
+There are three InfoBadge types that you can choose from - dot, icon, and numeric, as shown in order below.
+
+![Dot, icon, and numeric InfoBadges](images/infobadge-example2.png)
+
+### Dot InfoBadge
+The dot InfoBadge is a simple ellipsis with a default size of 4px by 4px. It has no border, and is never meant to hold text or anything else inside of it. 
+
+The dot InfoBadge should be used for general scenarios in which you want to guide the user’s focus towards the InfoBadge – i.e. indicating new content or updates are available.
+
+### Icon InfoBadge
+The icon InfoBadge is a 16px by 16px ellipsis that holds an icon inside of it. InfoBadge has an `IconSource` property which provides flexibility for the types of supported icons.  
+
+The icon InfoBadge should be used to send a quick message along with getting the user’s attention – i.e. alerting the user that something non-blocking in the application has gone wrong, an extra important update is available, or that something specific in the app is currently enabled (like a countdown timer going). 
+
+Note that if you'd like to use BitmapIcon for the `IconSource` of your InfoBadge, you are responsible for ensuring that the BitmapIcon fits inside of the InfoBadge (either by changing the size of the icon, or changing the size of the InfoBadge). 
+ 
+### Numeric InfoBadge 
+The numeric InfoBadge is also a 16px by 16px ellipsis that holds a number inside of it. The width of this ellipsis will automatically expand as the number grows to multiple digits, with a smooth animation. 
+
+The numeric InfoBadge should be used for showing that there are a specific number of items that need attention – i.e. new emails/messages.
```

Comment:
Should there be a corresponding `MaxValue` property which can be used to indicate if a number should switch to be like `9+` vs `10` if `MaxValue == 9`?

It could be set to infinity or null or something by default to indicate no overflow behavior?

Reply:
There was talk of having a MaxValue property but it was ultimately decided against based on precedent of other InfoBadges across different technologies, many of which show large numerical badges without cutting the developer off at a certain point. Another piece of the reasoning is that MaxValue is also something that could be implemented pretty easily by a developer in their own logic.